README.md

```diff
@@ -31,7 +33,8 @@ module "ssh_key_pair" {
 | `private_key_extension`      | ``             | Private key file extension (_e.g._ `.pem`)               | No        |
 | `public_key_extension`       | `.pub`         | Public key file extension (_e.g._ `.pub`)                | No        |
 | `chmod_command`              | `chmod 600 %v` | Template of the command executed on the private key file | Yes(Linux), No(Windows) |
-
+| `region`                     | `us-east-1`    | Region for AWS provider                                  | No        |
+| `only_generate_ssh_key`      | `false`        | If set to `true`, new SSH key pair will be created without importing the key pair into the AWS | No        |
```

Comment:
@aknysh what do you think about renaming to `import_key_pair`

Reply:
`import_key_pair` sounds better (and shorter)